casanovo/denovo/model_runner.py

```diff
@@ -255,6 +292,11 @@ def initialize_model(self, train: bool) -> None:
         if self.model_filename is None:
             # Train a model from scratch if no model file is provided.
             if train:
+                if db_search:
```

Comment:
**question:** I don't think this should happen within `if train` but rather as a separate check outside of that?

Reply:
Fixed